content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,77 @@
+ - `QuickTime (Mac)`: If you have a Mac, QuickTime Player has a built-in screen recording feature. You can record your screen and your webcam simultaneously.
+ - `Zoom (App/Web)`: Zoom's built-in recording feature makes it easy to capture your screen and webcam simultaneously. Share your slides, practice your one-minute pitch, and download the MP4 file for submission
+
+#### Slightly Advanced (Free/Open Source):
+ - `OBS Studio (Windows, Mac, Linux)`: OBS Studio is free and open-source, offering more control over your recording, including screen capture, webcam input, and basic audio mixing. While powerful, it might have a small learning curve.
+ - `VLC Media Player (Windows, Mac, Linux)`: VLC can also be used for basic recording. It's a versatile player that can capture video from your webcam. 
+
+#### User Friendly (Paid):
+ - `Canva`: offers a user-friendly platform with recording capabilities. You can easily create slides with your key points and record yourself presenting them. Canva also provides editing features to polish your video.
+
+## **How to Structure Your Video**
+
+Here's a simple way to approach your **one-minute** video:
+
+### **1. Start with the Basics**
+
+Introduce yourself and your idea. Pull from the abstract of your talk and expand slightly to explain what your talk is about.
+
+### **2. Share Why It Matters**
+
+Why is your idea important to you? Why does it excite you? People love to see genuine enthusiasm - it is contagious! Highlight why your topic could be interesting or helpful to the audience.
+
+### **3. Keep It Conversational**
+
+Talk as if you are **explaining to a friend,** not reading from a script. Keep it engaging and natural.
+
+### **4. Speak Clearly & With Energy**
+
+Passion is contagious! **Slow down a bit,** use your voice expressively, and **have fun with it.**
+
+### **5. Don't Worry About Perfection**
+
+Your video doesn't need to be flawless - just **authentic**. If selected, you will get speaker training!
+
+### **6. Humor and Creativity are Welcome**
+
+If your idea lends itself to something quirky or funny, don't shy away. Even if your topic seems niche, an engaging presentation can make it appealing to everyone.  But don't worry if you are not naturally funny either - this is only optional.
+
+## **Tips for Success**
+
+* **Pep Talk Yourself:** Before recording, remind yourself that you are great at what you do. Boosting your confidence will help you project positivity and ease.  
+* **Keep It Simple:** Your video doesn't need to be polished or perfect. Focus on your enthusiasm and message.  
+* **Check Out Examples:** Watch [other video abstracts](https://posit.co/blog/how-we-build-the-positconf-program/) for inspiration.
+
+## **Need Feedback?**
+
+If you would like input on your abstract or video, we are happy to help! Share a link to your draft (Google Drive or similar), and a reviewer will take a look.
```

Comment:
```suggestion
If you would like input on your abstract or video, we are happy to help! Share a link to your draft (Google Drive or similar), and we will assign a reviewer to take a look at it.  If you want feedback on a specific aspects of the video, please mention in the form, otherwise we will assume you're looking for general feedback.
```